truss/truss_config.py

```diff
@@ -241,6 +241,7 @@ class TrussConfig:
     bundled_packages_dir: str = DEFAULT_BUNDLED_PACKAGES_DIR
     external_package_dirs: List[str] = field(default_factory=list)
     live_reload: bool = False
+    enable_patches: bool = True
```

Comment:
Can we make this setting's name a bit more descriptive, like `apply_perf_patches`?

Reply:
They aren't all perf patches, in the case of OpenAI/Whisper - the model won't start with truss/b10cp

Reply:
does `apply_library_patches` work here?